### ISSUE TITLE
Same (°C) data despite period change for Berkeley Earth 'T' index 

### ISSUE THREAD

**Issue by sadielbartholomew** [IPCC-WG1/Atlas#98]
**Description of the problem**

I am exploring your very useful tool and noticed that in one case (other selections seem to work as they should) the exact same data is displayed despite my changing the configuration from the time period, namely when viewing the 'Berkeley Earth' observational dataset it is the same from '1961-2015' to '1980-2015', however physically of course one would expect the data to be different when extending the time range covered, whatever is selected for 'Season' (screenshots below show the example of the 'Annual' case).

See below for the details of the specific case where this occurs.

**To Reproduce**

Write below the steps to reproduce the problem:

1. Select 'Berkeley Earth' (under 'Observations' column) as the choice under 'Dataset';
2. Select 'Mean temperature (T)' under 'Variable'
3. For 'Quantity and Period' select 'Value (deg C)' (note that when the 'Trend' option is instead chosen, the data *does* change by toggling between the pair of periods, so this is key);
4. Select any choice under 'Season' - this changes the data but the data is still the same regardless of what of the two period options you select next.
5. Note toggling between the tow available periods, the data shown is identical (for any 'Season' choice above).

**Expected behavior**

I expect the data shown to be different when I click from one time period to the other, to reflect the extra years covered in the time period under consideration - statistically it would be highly unlikely the global data would be identical across different time ranges. Note I have also checked using an external tool (cf-python) that the generated netCDF are identical.

**Screenshots**

Using the example of the 'Annual' option for 'Season', note the data shown is the same despite the different selected period:

![atlas-2-bug](https://github.com/user-attachments/assets/46ed0b80-3baa-4e84-8ce5-b670d57de941)
![atlas-1-bug](https://github.com/user-attachments/assets/7d1d7695-edf6-437e-b16d-af54bc38a31a)

**Desktop (please complete the following information):**

 - OS: [e.g. iOS] (Linux) Manjaro
 - Browser: Firefox
 - Version [e.g. 22] 130.0 (64-bit), Firefox for Manjaro Linux and 6.5.13-7-MANJARO (64-bit) kernel version for Manjaro

**Additional context**

**Comment by jesusff** [comment]
Dear Sadie, thank you for reporting the problem. The Atlas team was able to reproduce it in the sense that visually they look very much alike. However, they are slightly different. The following links show a zoom on a region in Siberia for both periods:

1961-2015 https://interactive-atlas.ipcc.ch/wms?hatching=SIMPLE&DIM_period=1961-2015&DIM_time_filter=year&dataset=OBS-BerkeleyEarth-observations&layers=tas&DIM_baseline=preIndustrial&SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&FORMAT=image%2Fpng&TRANSPARENT=true&WIDTH=800&HEIGHT=600&CRS=EPSG%3A4326&STYLES=&BBOX=34.101563%2C32.842674%2C120.937500%2C56.559482

1980-2015 https://interactive-atlas.ipcc.ch/wms?hatching=SIMPLE&DIM_period=1980-2015&DIM_time_filter=year&dataset=OBS-BerkeleyEarth-observations&layers=tas&DIM_baseline=preIndustrial&SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&FORMAT=image%2Fpng&TRANSPARENT=true&WIDTH=800&HEIGHT=600&CRS=EPSG%3A4326&STYLES=&BBOX=34.101563%2C32.842674%2C120.937500%2C56.559482

The color range does not help, but there are small differences. We couldn't reproduce the identical netCDFs you mention, though . Do you mean those downloaded from the Interactive Atlas? We checked for differences and they also differ slightly (tenths of degrees in a range of absolute temperatures of more than 70 degrees):
![image](https://github.com/user-attachments/assets/066a4077-0a02-4d9e-af5b-e8747053898e)
![image](https://github.com/user-attachments/assets/1b1b097d-d1ce-49de-8cb7-35bfed043b12)

**Comment by sadielbartholomew** [comment]
Ah, thanks Jesus for reporting back. All good, in that case, and I will close this.

>  Do you mean those downloaded from the Interactive Atlas? We checked for differences and they also differ slightly (tenths of degrees in a range of absolute temperatures of more than 70 degrees):

Yes, I downloaded the netCDF files for each case, though I must have made some mistake with my field comparison against the two files. After looking now at my script it seems I was comparing the same field against itself, which naturally would have been registered as identical! Essentially I think this is a case of confirmation bias on my part, not being caregul enough to double check the script when I got a result that the two files were the same when actually comparing the same one against itself.
